index.js

```diff
@@ -50,4 +50,12 @@ types.forEach(function(type) {
   normalize[type] = normalize.bind(null, type);
 });
 
+normalize.date = function(value) {
```

Comment:
the most semantic name would be `dateOrTimestamp` but I guess I am fine with `date` unless someone else uses this and complains.